Assignment3/sketch.js

```diff
@@ -0,0 +1,73 @@
+/*
+ *
+ * Part - A : Refactor the following function into a one-liner
+ * 
+ */
+console.log("\nPart - A : Problem");
+
+var printName_old = (name) => {
```

Comment:
Please check the naming convention.

Reply:
Changed the names of functions adhering to naming conventions.